library/src/main/java/com/datatorrent/lib/partitioner/StatsAwareStatelessPartitioner.java

```diff
@@ -43,19 +43,23 @@
 
 /**
  * <p>
- * This does the partition of the operator based on the load. This partitioner doesn't copy the state during the repartitioning
- * Concrete classes should implement getLoad() function that calculates the load on the operator
+ * This does the partition of the operator based on the load. This partitioner
+ * doesn't copy the state during the repartitioning Concrete classes should
+ * implement getLoad() function that calculates the load on the operator
  * </p>
  *
  * <b>Properties</b>:<br>
- * <b>cooldownMillis:</b> The time for the operators to stabilize before next partitioning if required<br>
+ * <b>cooldownMillis:</b> The time for the operators to stabilize before next
+ * partitioning if required<br>
  * <br>
  *
- * @param <T> Operator type
+ * @param <T>
+ *          Operator type
  *
  * @since 2.1.0
  */
-public abstract class StatsAwareStatelessPartitioner<T extends Operator> implements StatsListener, Partitioner<T>, Serializable
+public abstract class StatsAwareStatelessPartitioner<T extends Operator> implements StatsListener, Partitioner<T>,
+    Serializable
```

Comment:
Few changes are related to new lines. It's hard to review,  Could you please revert those ?